client/signup/steps/intent/intent-screen.scss

```diff
@@ -0,0 +1,66 @@
+@import '@wordpress/base-styles/_breakpoints.scss';
+@import '@wordpress/base-styles/_mixins.scss';
+
+.intent-screen__cards {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin-top: -24px;
+	margin-bottom: 40px;
+	padding-left: 40px;
+	
+	@include break-small {
+		padding-left: 60px;
+	}
+}
+
+.intent-screen__card {
+	display: flex;
+	align-items: flex-start;
+	position: relative;
+	padding: 24px 0;
+	
+	@include break-small {
+		width: 342px;
+	}
+
+	@include break-medium {
+		width: 456px;
+	}
+
+	&-icon {
+		position: absolute;
```

Comment:
Also, this one is also my first implementation and I think it's not good to use absolute positioning now because it will have overlap between header and icon if the screen width is too small. So, maybe we can remove this and replace `border-top` with `::before`

```css
& + &::before {

element.style {
    height: 1px;
    width: calc(100% - 48px);
    margin-left: auto;
    background-color: #dcdcde;
}
```

Reply:
I added padding to prevent the icon from overlapping on small screens, I don't think that we need to add this element.
![shrink](https://user-images.githubusercontent.com/22446385/134996138-15a31ae3-3815-4646-9553-b440bc811309.gif)

Reply:
Well done 👍